pkg/db/db_deployments.go

```diff
@@ -381,28 +381,31 @@ func (h *DBHandler) DBSelectAllDeploymentsForApp(ctx context.Context, tx *sql.Tx
 	return h.processAllDeploymentRow(ctx, err, rows)
 }
 
-func (h *DBHandler) DBSelectAllDeploymentsForAppAtTimestamp(ctx context.Context, tx *sql.Tx, appName string, ts time.Time) (map[types.EnvName]int64, error) {
+func (h *DBHandler) DBSelectAllDeploymentsForAppAtTimestamp(ctx context.Context, tx *sql.Tx, appName string, ts time.Time) (map[types.EnvName]types.ReleaseNumbers, error) {
 	span, ctx := tracer.StartSpanFromContext(ctx, "DBSelectAllDeploymentsForAppAtTimestamp")
 	defer span.Finish()
 	query := h.AdaptQuery(`
 	SELECT
 		deployments_history.envName,
-		deployments_history.releaseVersion
+		deployments_history.releaseVersion,
+	    deployments_history.revision
```

Comment:
nitpick: indentation

```suggestion
		deployments_history.releaseVersion,
	        deployments_history.revision
```